src/BlockTypes/StockFilter.php

```diff
@@ -10,7 +10,9 @@ class StockFilter extends AbstractBlock {
 	 *
 	 * @var string
 	 */
-	protected $block_name = 'stock-filter';
+	protected $block_name               = 'stock-filter';
+	const STOCK_STATUS_QUERY_VAR        = 'filter_stock_status';
+	const STOCK_STATUS_QUERY_VAR_VALUES = array( 'instock', 'outofstock', 'onbackorder' );
```

Comment:
Should we use the stock statuses returned by `wc_get_product_stock_status_options()` instead of hardcoding them? This way this will be compatible with any extension hooking into the [`woocommerce_product_stock_status_options` filter](https://github.com/woocommerce/woocommerce/blob/9d96419520a712e691ce4fa8c172abb2248ba9f9/plugins/woocommerce/includes/wc-product-functions.php#L902-L909).

Reply:
Good point! I addressed the feedback with [d23d6a0](https://github.com/woocommerce/woocommerce-blocks/pull/7251/commits/d23d6a0d4f0c9d52e32abe27bb10a4b8dba5f7c2).